source/blog/2016-04-01-deadlock-investigation.html.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Debugging a deadlock in ManageIQ
+author: jrafanie
+date: 2016-04-01 18:20:08 UTC
+tags: developers, community, support
+comments: true
+published: true
+---
+
+In ManageIQ, we are seeing issues with our Node.js self service user interface.
```

Comment:
Node.js is an irrelevant detail here (is it even strictly true?); the buggy behaviour was always known to be in the Ruby code, so it seems distracting(?) to implicate something else by mere association.